src/core/blockchain/types/transaction.ts

```diff
@@ -137,5 +137,6 @@ export enum TransactionMessageText {
     CONTRACT_TX_NORMAL_NOT_ALLOWED = 'CONTRACT_TX_NORMAL_NOT_ALLOWED',
     WAITING_TX_CONFIRM = 'WAITING_TX_CONFIRM',
     WAITING_TX_CONFIRM_CANCEL = 'WAITING_TX_CONFIRM_CANCEL',
-    INSUFFICIENT_FUNDS_SOURCE_ACCOUNT = 'INSUFFICIENT_FUNDS_SOURCE_ACCOUNT'
+    INSUFFICIENT_FUNDS_SOURCE_ACCOUNT = 'INSUFFICIENT_FUNDS_SOURCE_ACCOUNT',
+    SENDING_VOTE = 'SENDING_VOTE'
```

Comment:
this might be confusing in celo where is vote. Maybe put 

`SENDING_GOVERNNANCE_VOTE`

Reply:
yeah, I agree, maybe `GOVERNANCE_VOTE` would be more suitable.